### PR TITLE
avoid pollution in the tmp directory when running the AlphaFold tests

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
bber0501u03a.bb2.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/d7d017eadcdb0092985a048052c582e3 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0212u15a.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) CPU E5-2640 v4 @ 2.40GHz (broadwell), 1 x NVIDIA Tesla K40m, 470.57.02, Python 3.6.8
See https://gist.github.com/4ad3c6747c3d1c99caf83dc83e2b13c6 for a full test report.

**Comment by boegel** [comment]
@branfosj Is this patch also relevant for older AlphaFold version (2.1.0, 2.0.0)?

**Comment by branfosj** [comment]
> @branfosj Is this patch also relevant for older AlphaFold version (2.1.0, 2.0.0)?

2.1.0 yes.

2.0.0 no. There was a single test, so this pollution problem will not occur. We could still add a similar patch to avoid reusing the directory anyway.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3900.accelgor.os - Linux RHEL 8.4, x86_64, AMD EPYC 7413 24-Core Processor (zen3), 1 x NVIDIA NVIDIA A100-SXM4-80GB, 510.39.01, Python 3.6.8
See https://gist.github.com/ae8fb8fb2284fdb24f28f047f67d2684 for a full test report.

File: easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.1_fix-alphafold-tests.patch [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Fix `run_alphafold_test.py` to use an unique directory per test to avoid pollution when the tests are run multiple times
+Patch by Simon Branford (University of Birmingham)
```
Comment (boegel) [inline easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.1_fix-alphafold-tests.patch:2]: @branfosj Let's refer to the upstream issue too here: https://github.com/deepmind/alphafold/issues/365

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.1_fix-alphafold-tests.patch (hunk 1 above)
Comment (branfosj) [reply easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.1_fix-alphafold-tests.patch:3]: done in e17e3b4

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3300.joltik.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz (cascadelake), 1 x NVIDIA Tesla V100-SXM2-32GB, 510.47.03, Python 3.6.8
See https://gist.github.com/e91489d6c52a69fe899cfc87402bca92 for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @branfosj!

**Pull request merged**